canonical-web-and-design/maas-ui
# feat(ui): split KVMResources into composable components

## Done

- Split `KVMResources` into just the subcomponents `KVMResourcesCard` and `KVMStorageCards`
- Built view components `LXDSingleResources` and `VirshResources` from the subcomponents
- Removed `OverallResourcesCard`

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the "Resources" tab of a LXD single host and check that it looks the same apart from the overall resources card is gone
- Go to the "Resources" tab of a virsh host and check that it looks the same

## Fixes

Fixes canonical-web-and-design/app-squad#301
Fixes canonical-web-and-design/app-squad#353
